MetadataExtractor/Formats/Exif/ExifDescriptorBase.cs

```diff
@@ -271,6 +273,25 @@ public string GetResolutionDescription()
                 "cm");
         }
 
+        [CanBeNull]
+        public string GetPageNumberDescription()
+        {
+            var values = Directory.GetInt32Array(ExifDirectoryBase.TagPageNumber);
```

Comment:
All the specs I have seen describe this as a short array. Exiftool has it typed as ushort. When feasible, I typically try to keep the same data types in .NET as the spec. Not every array type is implemented, so can fall back on the 'as' operator:

`var values = Directory.GetObject(ExifDirectoryBase.TagPageNumber) as ushort[];`

Obviously, the parsed datatype isn't always to spec so you would have to test whether this works all the time. Doing this in Java isn't always possible, so int32 isn't so bad if you're trying to match code.